diff --git a/libaccounts-glib/Makefile.am b/libaccounts-glib/Makefile.am
index ff4e25a..eb47ca8 100644
--- a/libaccounts-glib/Makefile.am
+++ b/libaccounts-glib/Makefile.am
@@ -9,7 +9,7 @@ libaccounts_glib_la_CPPFLAGS = \
 libaccounts_glib_la_CFLAGS = \
 	$(LIBACCOUNTS_CFLAGS) \
 	$(COVERAGE_CFLAGS) \
-	-Wall -Werror
+	-Wall -Werror -Wno-error=deprecated-declarations
 libaccounts_glib_la_LIBADD = \
 	$(LIBACCOUNTS_LIBS)
 libaccounts_glib_la_LDFLAGS = \
