From 9fb3c187e365d32cc607c00e688ada602edbea0e Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Tue, 21 Oct 2014 13:50:53 +0300
Subject: [PATCH] Add test directory path for test script

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 tests/tests.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/tests.xml.in b/tests/tests.xml.in
index 90f518b..92a7506 100644
--- a/tests/tests.xml.in
+++ b/tests/tests.xml.in
@@ -2,7 +2,7 @@
     <suite name="libaccounts-glib0-tests" domain="Accounts and SSO">
         <set description="libaccounts-glib0-tests:accounts-glib-tests" name="@TESTDIR@/accounts-glib-test.sh">
             <case description="libaccounts-glib-tests:accounts-glib-test.sh" name="accounts-glib-test.sh" type="Functional" level="Feature">
-                <step>(cd @TESTDIR@ &amp;&amp; TESTDATADIR=@TESTDATADIR@ ./accounts-glib-test.sh)</step>
+                <step>(cd @TESTDIR@ &amp;&amp; TESTDIR=$(/bin/pwd) TESTDATADIR=@TESTDATADIR@ ./accounts-glib-test.sh)</step>
             </case>
             <environments><scratchbox>true</scratchbox>
             <hardware>true</hardware></environments>
-- 
1.9.1

