From 89159e10de8071079ea2758a30bce04a8a7522ba Mon Sep 17 00:00:00 2001
From: John Brooks <john.brooks@jollamobile.com>
Date: Mon, 28 Oct 2013 05:48:49 -0600
Subject: [PATCH 5/5] Remove use of function only available in check 0.9.10

---
 tests/check_ag.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/check_ag.c b/tests/check_ag.c
index 7507906..2e9a267 100644
--- a/tests/check_ag.c
+++ b/tests/check_ag.c
@@ -1517,7 +1517,7 @@ START_TEST(test_service)
     ag_account_set_display_name (account, display_name);
 
     ag_account_select_service (account, service);
-    ck_assert_ptr_eq (ag_account_get_selected_service (account), service);
+    ck_assert (ag_account_get_selected_service (account) == service);
 
     /* test getting default setting from template */
     g_value_init (&value, G_TYPE_INT);
-- 
1.7.9.5

